### ISSUE TITLE
Integrate anvil into mocktimism

### ISSUE THREAD

**Issue by roninjin10** [ethereum-optimism/mocktimism#50]
This is a subtask of https://github.com/ethereum-optimism/mocktimism/issues/9

Depends on following issues
https://github.com/ethereum-optimism/mocktimism/issues/5
https://github.com/ethereum-optimism/mocktimism/issues/4
https://github.com/ethereum-optimism/mocktimism/issues/49

## Description

Once anvil is ready we want to replace geth and opgeth with anvil and opanvil. We should be able to replace l1 with anvil before op-anvil is ready

This does not include hooking anvil up to the mocktimism toml configuration. For now anvil should be hardcoded. This work is captured in https://github.com/ethereum-optimism/mocktimism/issues/28

## Tasks

- [ ] Replace l1 node with anvil
- [ ] Replace l2 node with anvil with optimism flag passed in
- [ ] Write a simple integration test between op-node and anvil

**Comment by nickbalestra** [comment]
Merged into #9 as its currently a duplicate
